.github/actions/build-avr/install.sh

```diff
@@ -0,0 +1,21 @@
+# Exit immediately if a command exits with a non-zero status.
```

Comment:
This script appears to replicate the steps taken in the `Dockerfile`. I don't see it called anywhere, and the name is very close to conflicting with the Arduino `install.sh` script. What am I missing?

Reply:
It was intended for local development, but really we can just use the Dockerfile, so it's removed now.